c3d-apply/workflow/scripts/seg_to_fcsv.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import nibabel as nib
+import numpy as np
+
+
+def compute_centroid(seg, affine, fid):
+    # Get slice indices matching fiducial
+    x, y, z = np.where(seg == fid)
+
+    # Compute centroid
+    # Better performance than np.mean()
+    centroid = [np.sum(x) / len(x), np.sum(y) / len(y), np.sum(z) / len(z)]
+
+    # Return centroid in world coordinates
+    return affine[:3, :3].dot(centroid) + affine[:3, 3]
+
+
+### MAIN ###
```

Comment:
Just as a general good practice, I would wrap this in a `main()` function, and call it at the bottom with `if __name__ == "__main__":`

Reply:
I agree - wasn't sure how snakemake handled python scripts. Easy change to make.

Reply:
Made the change in [945a536](https://github.com/afids/afids-auto/pull/4/commits/945a536bd57e88c57034eab5355daf30d0b2e5ee), not entirely sure if it works with snakemake (couldn't find any concrete examples), but worth a shot. Easy to revert if it doesn't.